source/features/show-open-prs-of-forks.tsx

```diff
@@ -43,11 +43,11 @@ const countPRs = cache.function(async (forkedRepo: string): Promise<[number, num
 }, {
 	maxAge: 1 / 2, // Stale after 12 hours
 	staleWhileRevalidate: 2,
-	cacheKey: ([forkedRepo]): string => 'prs-on-forked-repo:' + forkedRepo
+	cacheKey: ([forkedRepo]): string => 'prs-on-forked-repo:' + forkedRepo + ':' + getRepoURL()
 });
 
 async function getPRs(): Promise<[number, string] | []> {
-	await elementReady('.repohead + *'); // Wait for the tab bar to be loaded
+	await elementReady('.pagehead + *, .UnderlineNav-body + *'); // Wait for the tab bar to be loaded
```

Comment:
Why does this also changes `repohead`?

Reply:
We need to wait for bar to load. Repohead does not exist anymore.

Reply:
> Isn't that part just for GHE now?

I dont remember I copied from `bug-tab`

Reply:
This exists in GHE with the new selector. If there are specific concerns, @ me and I can check.

_Originally posted by @busches in https://github.com/_render_node/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkMjg0NzQwMzA4OnYy/pull_request_review_threads/discussion_

Reply:
Uh that link did not work. https://github.com/sindresorhus/refined-github/pull/3361#discussion_r456548382